nodejs/week3/warmUp/index.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.json());
+
+// Calculator using params
+app.get('/calculator/:param', (req, res) => {
+	const { param } = req.params;
+	const { firstParam } = req.query;
+	const { secondParam } = req.query;
+	const allNumbers = [];
+	function calculateAccordingToParam() {
+		if (param === 'addition') {
+			const valueOfAddition = allNumbers.reduce((acc, value) => (acc += value));
+			return `Answer of addition: ${valueOfAddition}`;
+		} else if (param === 'subtraction') {
+			const valueOfSubstraction = allNumbers.reduce((acc, value) => (acc -= value));
+			return `Answer of substraction: ${valueOfSubstraction}`;
+		} else if (param === 'multiplication') {
+			const valueOfMultiplication = allNumbers.reduce((acc, value) => (acc *= value));
+			return `Answer of multiplication: ${valueOfMultiplication}`;
+		} else if (param === 'division') {
+			const valueOfDivision = allNumbers.reduce((acc, value) => (acc /= value));
+			return `Answer of Division: ${valueOfDivision}`;
+		}
+	}
+	
+	if (typeof firstParam === 'object' && typeof secondParam === 'object') {
+		firstParam.forEach(nums1 => allNumbers.push(parseInt(nums1)));
```

Comment:
Hmm why would firstParam be an array?

Reply:
Its basically, I got it when I  console.log firstParam & secondParam.
If I gave more than a single number to firstParam & secondParam, I got an array, so thats how i ended up with these conditions.